### PR TITLE
re-edit for JS-disabled support

### LINKED ISSUE #85: Hide "copy to clipboard" button for JavaScript-disabled clients
This will likely be done by adding the button using JavaScript

### PR DESCRIPTION
**Pull request by ahmetlii**
Closes #85

### CODE REVIEW

File: src/SpecialRequestAccount.php [hunk 1]
```diff
@@ -217,7 +217,7 @@ function usernameAndVerificationArea(&$session, $request) {
 		$form .= Html::element(
 			'button',
 			[
-				'class' => 'mw-scratch-confirmaccount-click-copy',
+				'class' => 'jsonly',
```
Comment (Kenny2github) [inline src/SpecialRequestAccount.php:220]:
```suggestion
				'class' => 'mw-scratch-confirmaccount-click-copy jsonly',
```

File: resources/main.css [hunk 2]
```diff
@@ -48,6 +48,9 @@
 	color: red;
 	font-weight: bold;
 }
-.mw-scratch-confirmaccount-click-copy {
+#mw-scratch-confirmaccount-click-copy {
```
Comment (Kenny2github) [inline resources/main.css:51]: Undo this in conjunction with my other comment please.

**Review by Kenny2github — CHANGES_REQUESTED** [review]

File: resources/main.css (hunk 2 above)
Comment (ahmetlii) [reply resources/main.css:51]: ok

File: resources/main.css [hunk 3]
```diff
@@ -48,6 +48,9 @@
...
 	font-weight: normal;
 }
+.jsonly noscript style{
```
Comment (jacob-g) [inline resources/main.css:54]: Is this even possible?

**Review by jacob-g — CHANGES_REQUESTED** [review]

File: resources/main.css (hunk 3 above)
Comment (ahmetlii) [reply resources/main.css:54]: It is (because we're tagging it with priority as far as I know).

Comment (jacob-g) [reply resources/main.css:54]: From what I can tell this isn't. You will need to inject this CSS into the head using the appropriate MediaWiki libraries.

Comment (jacob-g) [reply resources/main.css:54]: I should say what the semantics of this rule do:
They apply the rule to a `<style>` tag inside a `<noscript>` tag inside a tag with a class of jsonly. You shouldn't apply CSS rules directly to a `<style>` at all tag. Also, please test code before submitting a PR.

**Review by jacob-g — APPROVED** [review]

**Pull request merged**